sdk/tables/perf-tests/data-tables/test/index.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { PerfStressProgram, selectPerfStressTest } from "@azure/test-utils-perfstress";
+import { CreateSimpleEntityTest } from "./createSimpleEntity.spec";
+
+// Expects the .env file at the same level
+import * as dotenv from "dotenv";
+import { CreateEntityBatchTest } from "./createEntityBatch.spec";
+dotenv.config();
+
+console.log("=== Starting the perfStress test ===");
+
+const perfStressProgram = new PerfStressProgram(selectPerfStressTest([CreateSimpleEntityTest, CreateEntityBatchTest]));
```

Comment:
Can you also check with other languages and make sure that we are all in sync on the scenarios for the perf tests?

Reply:
Yup, I'm aligning with other languages on the scenarios and entities being created and queired